packages/data-point/lib/entity-types/entity-request/resolve.test.js

```diff
@@ -175,6 +175,13 @@ describe('getRequestOptions', () => {
 })
 
 describe('resolveRequest', () => {
+  let consoleInfo
+  beforeAll(() => {
+    consoleInfo = console.info
+  })
+  afterEach(() => {
+    console.info = consoleInfo
+  })
```

Comment:
Sorry, I'm confused by this. Are we just mocking `console.info` and resetting before each test? Would this be possible/equivalent?

```javascript
describe('resolveRequest', () => {
  const consoleInfoMock = jest.fn()
  console.info = consoleInfoMock
  beforeEach(() => {
    consoleInfoMock.mockClear()
  })
  .
  .
  .
  expect(consoleInfoMock).toBeCalled()
```

Not requesting a change exactly, just want to be sure I'm reading it right 😄

Reply:
yes would be but because the tests as async, expects can get out of sync, already seen than happen to me before... can drive you crazy

Reply:
OK, thanks for confirming.